demos/interactive_face_detection_demo/cpp_gapi/main.cpp

```diff
@@ -456,10 +456,10 @@ int main(int argc, char *argv[]) {
                 throw std::invalid_argument(msg.str());
             }
 
-            timer.start("total");
+            auto startTime = std::chrono::steady_clock::now();
             stream.start();
-            while (stream.pull(cv::GRunArgsP(out_vector))) {
-                if (!FLAGS_no_show && !FLAGS_m_em.empty() && !FLAGS_no_show_emotion_bar) {
+            while (startTime = std::chrono::steady_clock::now(), stream.pull(cv::GRunArgsP(out_vector))) {
```

Comment:
Why add `startTime = std::chrono::steady_clock::now()` as condition?

Reply:
It's not a condition, it's just the way to measure time at the start of cycle, before` stream.pull(cv::GRunArgsP(out_vector)`.

Reply:
Changed.